test/java/seedu/addressbook/commands/SortCommandTest.java

```diff
@@ -0,0 +1,89 @@
+package seedu.addressbook.commands;
+
+import org.junit.Before;
+import org.junit.Test;
+import seedu.addressbook.data.AddressBook;
+import seedu.addressbook.data.person.Address;
+import seedu.addressbook.data.person.Email;
+import seedu.addressbook.data.person.Name;
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.person.Phone;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+import seedu.addressbook.data.person.UniquePersonList;
+import seedu.addressbook.data.tag.Tag;
+import seedu.addressbook.data.tag.UniqueTagList;
+
+import java.util.Collections;
+import java.util.List;
+
+import static org.junit.Assert.*;
```

Comment:
Do not import using `*`. Instead, import the classes individually.

Reply:
Furthermore, static imports should be ordered before non-static ones for consistency.